test/utils/shallowEquality.spec.js

```diff
@@ -0,0 +1,138 @@
+import expect from 'expect';
+import shallowEqualScalar from '../../src/utils/shallowEqualScalar';
+import shallowEqual from '../../src/utils/shallowEqual';
+
+describe('Utils', () => {
+  // More info: https://github.com/gaearon/redux/pull/75#issuecomment-111635748
+  describe('shallowEqualScalar', () => {
+    it('returns true if both arguments are the same object referentially', () => {
+      const o = {a: 1, b: 2};
```

Comment:
Please tweak the style to use spaces inside object definitions. (`{a : 1}` => `{ a: 1 }`). This doesn't concern arrays, just objects.